addonscript/tox.ini

```diff
@@ -2,14 +2,14 @@
 envlist = docker
 
 [testenv:docker]
-whitelist_externals=docker
+allowlist_externals=docker
 deps =
 usedevelop = false
 depends =
 skip_install = true
 commands =
-    docker build --build-arg PYTHON_VERSION=3.9.7 -t addonscript-{envname}-py39-test -f Dockerfile.test .
-    docker run --rm -v {toxinidir}:/app -v addonscript-{envname}-py39-tox:/app/.tox addonscript-{envname}-py39-test py39
+    docker build --build-arg PYTHON_VERSION=3.11.9 -t addonscript-{envname}-py311-test -f Dockerfile.test .
+    docker run --rm -v {toxinidir}:/app -v addonscript-{envname}-py311-tox:/app/.tox addonscript-{envname}-py311-test py311
```

Comment:
Do we use these docker envs for testing anywhere?

Reply:
I sometimes use them for local testing, at least.